### ISSUE TITLE
Small translation updates required

### ISSUE THREAD

**Issue by ar0ne** [MovingBlocks/Terasology#2921]
Hi All,

This issue related PR #2920, where we decided to change part of greeting phrase ("enter-username-message") 
**From**: Would you like to use a non-default username?
**To**: Would you like to update your username?

You can see it when first visit of Join/Host game menu. 
At present we need small changes in our translations for next languages:

- [ ] cs (Czech)
- [ ] fa (Farsi)
- [ ] fr (French)
- [ ] hi (Hindi)
- [ ] id (Indonesian)
- [ ] ko (Korean)
- [ ] nl (Dutch)
- [ ] pr (Pirate) :D
- [x] Russian (recently checked by Translator5)
- [x] German (recently checked by Translator5)

I tried to mark this word as "fuzzy" in weblate (http://translate.terasology.org/projects/terasology/menu/), but it looks like we don't have all requested languages there. Correct me if I'm wrong, guys?

**Comment by Translator5** [comment]
I now fully re-translated the German translation manually.
(With open Terasology on the one desk and Weblate on the other one.)
Now there are much more context-related phrases!
*As soon as GitHub will recognize them, I will create a new Pull request!

Based on the German translation I will try to do the Russian translation soon.
The current Russian translation is full of false-context translations or totally false ones!
*Note to all translators;
Do not use just the english-source, but also use other languages to get the best translation!

**Comment by Cervator** [comment]
Bump - merged the first PR and Weblate has refreshed now. 

@Translator5 could you please double check on Russian? I had to resolve some conflicts manually and a few might have gone with a worse option. Hard for me to tell which is better :-)

There are other known issues where some languages aren't showing up in Weblate right. Not sure at present how to fix, need somebody able to spend some time poking at the server itself, maybe reset/refresh something else.

Changing the bullet list to a checkbox list and marking off German and Russian, hope that's accurate as of now.

**Comment by Translator5** [comment]
Some issues, belonging to the translation;
- we need separate name-strings for the "Player-Name:"-input and the "Behaviour"- as well as the "NUI-Editors"-title-input - because of translation issues!
- we should change the Settings in the NUI-Editors from "disable XYZ" into "enable XYZ" or "toggle XYZ" cause of translation issues (it looks better if settings just take one text-line, instead of 2 lines)

**Comment by Cervator** [comment]
Thanks for the spam report - yeah I noticed that too a few days ago but didn't have time to figure out what to do about it. Just now I spent a bit of time digging around and found a way to bulk delete all existing suggestions (including maybe 10 or so valid ones, sadly) so Weblate is clean now.

I didn't find an obvious way to simply disallow anonymous suggestions, seems like it is either turn them off entirely or enable a full ACL and manage everything via specific user permissions. So I simply turned them off. I don't think we really check for and apply suggestions anyway.

**Comment by Translator5** [comment]
> I don't think we really check for and apply suggestions anyway.
I personally use them, its helpful. E.g. for other possible translations, maybe some translations do not work cause of text-field size, context, multiple-context possible translations etc.

**Comment by Cervator** [comment]
I'd be happy to add them back in when we have some web master type capacity to maintain the system better. Like setting up a sensible user permissions setup that'll still make it really easy to start translating but harder to spam :-)

**Comment by dee6600** [comment]
i m willing to do in hindi

**Comment by reirep** [comment]
If nobody is on it yet, I can help with the French part

**Comment by fortiseven** [comment]
Hello there, I've been recently tempted to translate the game in Albanian, I finished my translation, but I've got a problem. I want to test the translation I've made, but for some unknown reason, the game doesn't detect the language I've added in the game's settings. I've put the .lang file here engine/src/main/resources/assets/i18n and the flag png here  engine/src/main/resources/assets/textures/ui/icons/flags. I've checked the Translate wiki again to make sure I didn't miss something, but it doesn't say anything in particular, it just says I need to add those two files in the right areas. Any ideas ?

I use windows 10, and I use IntelliJ.
